digits/templates/datasets/images/generic/summary.html

```diff
@@ -21,6 +21,7 @@
         <li><b>Image Count</b> - {{task.image_count}}</li>
         <li><b>Image Dimensions</b> -
         {{task.image_width}}x{{task.image_height}}x{{task.image_channels}}</li>
+        <li><b>Image bit-depth</b></li>
```

Comment:
I added that because it's part of UI selection.  However, you are right about the usefulness.  Old dataset did not have this info and the summary showed 'Image bit-depth' but nothing below it.  Maybe it causes more confusing.